test/run-pass/intrin/typeid_for/BUILD

```
@@ -0,0 +1,3 @@
+load("//test:tests.bzl", "generate_tests")
+
+generate_tests(skip_files = [])
```

You can drop the `skip_files` argument if it is empty.
```suggestion
generate_tests()
```